ui/admin/app/templates/scopes/scope/workers/worker/tags.hbs

```diff
@@ -0,0 +1,62 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.worker.tags.title_plural')}}
+<Breadcrumbs::Item
+  @text={{t 'resources.worker.tags.title_plural'}}
+  @route='scopes.scope.workers.worker.tags'
+/>
+
+<Rose::Layout::Page as |page|>
+
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <Workers::Worker::Header @model={{@model}} />
+  </page.header>
+
+  <page.navigation>
+    <Workers::Worker::Nav />
+  </page.navigation>
+
+  <page.actions>
+    <Workers::Worker::Actions
+      @model={{@model}}
+      @delete={{this.workers.delete}}
+    />
+  </page.actions>
+
+  <page.body>
+    {{#if @model.allTags}}
+      <Hds::Table
+        @model={{@model.allTags}}
+        @columns={{array
+          (hash label=(t 'resources.worker.table.key'))
+          (hash label=(t 'resources.worker.table.value'))
+          (hash label=(t 'resources.worker.table.type'))
+        }}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              <Hds::Text::Code @tag='pre'>{{B.data.key}}</Hds::Text::Code>
+            </B.Td>
+            <B.Td>
+              <Hds::Text::Code @tag='pre'>{{B.data.value}}</Hds::Text::Code>
+            </B.Td>
+            <B.Td>
+              <Hds::Badge @text={{B.data.type}} />
+            </B.Td>
+          </B.Tr>
+        </:body>
+      </Hds::Table>
+    {{/if}}
```

Comment:
non-blocking: are we going to add the empty app state for this tab in a later PR?

Reply:
Absolutely. I was going to add that when I created the "create new tags" route and view.